exonum/CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ All notable changes to this project will be documented in this file. The project
   around a type (e.g., `struct QuirkyI32Key(i32)`) and implement `StorageKey` for it
   using big endian encoding. Then, use the wrapper instead of the int type in indices.
   See the unit tests for `StorageKey` for an example. (#443)
+  
+### New features
+- `StorageKey` and `StorageValue` are implemented for `SystemTime`. (#456)
```

Comment:
I propose to add the word "traits": `StorageKey` and `StorageValue` _traits_ are...